packages/react-native-payments-example/src/app/app.tsx

```diff
@@ -17,11 +17,11 @@ export const App = (): JSX.Element => {
     const [response, setResponse] = useState<object>();
     const [isWalletAvailable, setIsWalletAvailable] = useState(false);
 
-    const createPaymentRequest = (): PaymentRequest => {
+    const createPaymentRequest = (paymentDetails?: PaymentDetailsInit): PaymentRequest => {
```

Comment:
I suggest instead of making this argument optional and passing the `defaultPaymentDetails` we can make this mandatory and pass the `defaultPaymentDetails` inside the `handlePayWithAbort`